examples/forall-param-reductions.cpp

```diff
@@ -41,118 +41,124 @@ constexpr int HIP_BLOCK_SIZE = 256;
 constexpr int SYCL_BLOCK_SIZE = 256;
 #endif
 
-int main(int RAJA_UNUSED_ARG(argc), char** RAJA_UNUSED_ARG(argv[]))
+int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
 {
 
   std::cout << "\n\nRAJA reductions example...\n";
 
   // _reductions_array_init_start
-//
-// Define array length
-//
+  //
+  // Define array length
+  //
   constexpr int N = 1000000;
 
-//
-// Allocate array data and initialize data to alternating sequence of 1, -1.
-//
+  //
+  // Allocate array data and initialize data to alternating sequence of 1, -1.
+  //
   RAJA::resources::Host host_res;
-  int* a = host_res.allocate<int>(N);
+  int *a = host_res.allocate<int>(N);
```

Comment:
Hmmm. It seems like it's being inconsistent. I think we should try to get it all the same way. I prefer 'int* a;' for example, but others do 'int *a;'.  I don't really care as long as the formatter makes it consistent.